pom.xml

```diff
@@ -115,9 +155,9 @@
                 <configuration>
                     <configFile>${project.basedir}/src/main/resources/moco-code-conv.xml</configFile>
                     <overrideConfigCompilerVersion>true</overrideConfigCompilerVersion>
-                    <compilerSource>1.7</compilerSource>
-                    <compilerCompliance>1.7</compilerCompliance>
-                    <compilerTargetPlatform>1.7</compilerTargetPlatform>
+                    <compilerSource>1.8</compilerSource>
```

Comment:
I think we need a JDK version bump in the `.travis-ci.yml` aswell, to make Travis happy.